index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
![image](https://user-images.githubusercontent.com/85732251/205759167-c0f92c47-9f84-46c1-ae95-680d7a5cf969.png)

Woa, what a nice project!
There are some issues that you still need to work on to go to the next project but you are almost there!

You documented your work professionally. ✔
GitHub flow not correctly used. ✔

Here are few changes you need to make 

Put the linter file in the root folder. ❌ 
Delete the whitespace. ❌